docker/alpine.dockerfile

```diff
@@ -12,6 +12,7 @@ RUN apk update \
         protobuf-dev \
         grpc
 
+ENV IsAlpine=true
```

Comment:
This is revert, from one of the previous commits.
Needed to detect type of the linux under the tests.

Reply:
Are you sure. I think it was unused. Can you doublecheck please?

Reply:
I removed it here: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1402/files